internal/auth0/tenant/resource.go

```diff
@@ -102,12 +102,12 @@ func NewResource() *schema.Resource {
 				Optional: true,
 				Computed: true,
 				MaxItems: 1,
-				Description: "Configuration settings for error pages. This attribute has been deprecated in favour " +
-					"of the newly introduced `auth0_pages` resource and it will be removed in a future version. " +
-					"Check the [MIGRATION_GUIDE](https://github.com/auth0/terraform-provider-auth0/blob/main/MIGRATION_GUIDE.md) for more info.",
-				Deprecated: "This attribute has been deprecated in favour of the newly introduced `auth0_pages` " +
-					"resource and it will be removed in a future version." +
-					"Check the [MIGRATION_GUIDE](https://github.com/auth0/terraform-provider-auth0/blob/main/MIGRATION_GUIDE.md) for more info.",
+				Description: "Configuration settings for error pages. This attribute is deprecated in favour " +
```

Comment:
```suggestion
				Description: "Configuration settings for error pages. This attribute is deprecated in favor " +
```
The guidelines from the Docs team suggest to use American English.